src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -208,7 +219,7 @@ Key Point: An <code>AudioContext</code> must be created or resumed after the
 document received a user gesture to enable audio playback.
 
 If you create your <code>AudioContext</code> on page load, you’ll
-have to call <code>resume()</code> later when user interacts with the page
+have to call <code>resume()</code> later after user interacted with the page
```

Comment:
Should be "have to call resume() at some time after the user has interacted with the page"

Reply:
Done.